statusHandler/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -273,10 +273,15 @@ func (wr *WidgetsRender) prepareListWithLogsForDisplay() {
 	wr.lLog.WrapText = true
 }
 
-func (wr *WidgetsRender) prepareLogLines(logData []string, size int) []string {
+func (wr *WidgetsRender) prepareLogLines(logData []string, maxSize int) []string {
 	logDataLen := len(logData)
-	if logDataLen > size {
-		return logData[logDataLen-size : logDataLen]
+	maxSize = maxSize - 2 // decrease 2 units as the total maxSize of the log list includes also the header and the footer
+	if maxSize < 0 {
```

Comment:
You can change condition here.  maxSize <= 0 you can  return  an empty slice of strings (return []string{}). Because when maxSize its equals with 0 you will return logData[(logDataLen - 0):logDataLen] and this means and empty slice.

Reply:
done